README.md

```
@@ -59,7 +59,7 @@ Furthermore, I wanted to make Arbigent accessible to QA engineers by offering a
     *   **Accessibility-Independent:** Provides annotated screenshots to assist AI in understanding UIs that lack accessibility information.
 *   **Cost Savings:**
     *   **Open Source:** Free to use, modify, and distribute, eliminating licensing costs.
-    *   **Efficient Model Usage:** Compatible with cost-effective models like `GPT-4o mini`, reducing operational expenses.
+    *   **Efficient Model Usage:** Compatible with cost-effective models like `GPT-4.1`, reducing operational expenses.
```

markdownlint-cli2 (0.17.2)

Unordered list indentation
Expected: 2; Actual: 4

(MD007, ul-indent)